service/reducers/requests/helpers.js

```diff
@@ -25,7 +25,8 @@ export function handleRequest(REQUEST, SUCCESS, FAILURE, state, action) {
         };
     case FAILURE: {
         let error = action.error;
-        if (error instanceof Error) {
+
+        if (error instanceof Error && !error.hasOwnProperty('id')) {
```

Comment:
I believe this line should take care of that, it checks to see if we have an ```id``` property on the error object.

https://github.com/mattermost/mattermost-mobile/blob/master/app/components/error_text.js#L22